src/state/helpers.ts

```diff
@@ -94,7 +100,9 @@ export const getEnabledSwappers = (
     [SwapperName.Cetus]:
       CetusSwap && (!isCrossAccountTrade || isCrossAccountTradeSupported(SwapperName.Cetus)),
     [SwapperName.Sunio]:
-      SunioSwap && (!isCrossAccountTrade || isCrossAccountTradeSupported(SwapperName.Sunio)),
+      SunioSwap &&
+      (!isCrossAccountTrade || isCrossAccountTradeSupported(SwapperName.Sunio)) &&
+      !isLedgerTronSell,
```

Comment:
Payload too large for Ledger device 

<img width="499" height="177" alt="Image" src="https://github.com/user-attachments/assets/4945600c-0014-4da8-8f2f-cc8deed4bacb" />